Core/GCTestCase.h

```diff
@@ -79,6 +79,9 @@
 @interface GCMultipleCommitsRepositoryTests : GCMultipleCommitsRepositoryTestCase
 @end
 
+@interface GCUniqueCommitCollectionTests : GCMultipleCommitsRepositoryTestCase
```

Comment:
That was my first approach. But then I found out that I have to call methods like `testCommitsRepositoryAddCommit` and so forth. Because `testAdd` makes no sense on the class named `GCMultipleCommitsRepositoryTestCase`, it is misleading.

Then I decided to make this class and I like how it actually looks in XCode testing section.
![gctestcase m 2015-08-21 08-10-45](https://cloud.githubusercontent.com/assets/466427/9402310/27d850d6-47dc-11e5-833c-5f4c72e3bd3d.png)